commands/bot.go

```diff
@@ -216,22 +217,27 @@ func botListCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 
 func botEnableCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	users := getUsersFromUserArgs(c, args)
+
+	var result *multierror.Error
+
 	for i, user := range users {
 		if user == nil {
 			printer.PrintError(fmt.Sprintf("can't find user '%v'", args[i]))
+			result = multierror.Append(result, fmt.Errorf("can't find user '%v'", args[i]))
```

Comment:
Please use `%q` to quote strings
```suggestion
			result = multierror.Append(result, fmt.Errorf("can't find user %q", args[i]))
```